public/clew/clew.nocache.js

```diff
@@ -0,0 +1,26 @@
+function clew(){var O='bootstrap',P='begin',Q='gwt.codesvr.clew=',R='gwt.codesvr=',S='clew',T='startup',U='DUMMY',V=0,W=1,X='iframe',Y='javascript:""',Z='position:absolute; width:0; height:0; border:none; left: -1000px;',$=' top: -1000px;',_='CSS1Compat',ab='<!doctype html>',bb='',cb='<html><head><\/head><body><\/body><\/html>',db='undefined',eb='DOMContentLoaded',fb=50,gb='Chrome',hb='eval("',ib='");',jb='script',kb='javascript',lb='moduleStartup',mb='moduleRequested',nb='Failed to load ',ob='head',pb='meta',qb='name',rb='clew::',sb='::',tb='gwt:property',ub='content',vb='=',wb='gwt:onPropertyErrorFn',xb='Bad handler "',yb='" for "gwt:onPropertyErrorFn"',zb='gwt:onLoadErrorFn',Ab='" for "gwt:onLoadErrorFn"',Bb='#',Cb='?',Db='/',Eb='img',Fb='clear.cache.gif',Gb='baseUrl',Hb='clew.nocache.js',Ib='base',Jb='//',Kb='user.agent',Lb='webkit',Mb='safari',Nb='msie',Ob=10,Pb=11,Qb='ie10',Rb=9,Sb='ie9',Tb=8,Ub='ie8',Vb='gecko',Wb='gecko1_8',Xb=2,Yb=3,Zb=4,$b='selectingPermutation',_b='clew.devmode.js',ac='00D440B2CFB7AB4335A0297C4E082D74',bc='02B111B78E448F55787C58C99814128F',cc='855B4A25C789D5AA39C9643C79DA3447',dc='E223BB3561A2B5524E1DC62EBDB55080',ec='F8406A36B97641EB209251D591DF2652',fc=':',gc='.cache.js',hc='loadExternalRefs',ic='end',jc='http:',kc='file:',lc='_gwt_dummy_',mc='__gwtDevModeHook:clew',nc='Ignoring non-whitelisted Dev Mode URL: ',oc=':moduleBase';var o=window;var p=document;r(O,P);function q(){var a=o.location.search;return a.indexOf(Q)!=-1||a.indexOf(R)!=-1}
+function r(a,b){if(o.__gwtStatsEvent){o.__gwtStatsEvent({moduleName:S,sessionId:o.__gwtStatsSessionId,subSystem:T,evtGroup:a,millis:(new Date).getTime(),type:b})}}
+clew.__sendStats=r;clew.__moduleName=S;clew.__errFn=null;clew.__moduleBase=U;clew.__softPermutationId=V;clew.__computePropValue=null;clew.__getPropMap=null;clew.__installRunAsyncCode=function(){};clew.__gwtStartLoadingFragment=function(){return null};clew.__gwt_isKnownPropertyValue=function(){return false};clew.__gwt_getMetaProperty=function(){return null};var s=null;var t=o.__gwt_activeModules=o.__gwt_activeModules||{};t[S]={moduleName:S};clew.__moduleStartupDone=function(e){var f=t[S].bindings;t[S].bindings=function(){var a=f?f():{};var b=e[clew.__softPermutationId];for(var c=V;c<b.length;c++){var d=b[c];a[d[V]]=d[W]}return a}};var u;function v(){w();return u}
```

Comment:
Line is too long.<br>Missing semicolon.<br>Identifier '__gwt_isKnownPropertyValue' is not in camel case.<br>Identifier '__gwt_getMetaProperty' is not in camel case.<br>Identifier '__gwt_activeModules' is not in camel case.